### PR TITLE
chore(*): Fix eslint configs

### PR DESCRIPTION
**Pull request by imagoiq**

### CODE REVIEW

File: packages/components/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:@stencil-community/recommended',
+  ],
+  overrides: [],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    tsconfigRootDir: __dirname,
```
Comment (imagoiq) [inline packages/components/.eslintrc.js:17]: Convert to js to be able to use `__dirname`

**Review by alizedebray — APPROVED** [review]

**Review by gfellerph — CHANGES_REQUESTED** [review]
Looking good so far

File: packages/components/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (imagoiq) [reply packages/components/.eslintrc.js:1]: Unfortunately, it's not as easy as that. I think the error only appears in your IDE, but not the tools. We need to make a lot of changes so I created another ticket for that: https://github.com/swisspost/design-system/issues/2167

File: packages/internet-header/.eslintignore [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.stencil
+loader
+www
+dist
+node_modules
+cypress
+stencil.config.ts
+cypress.config.js
+jest.config.js
```
[the remark this answers is not in the document]
Comment (imagoiq) [reply packages/internet-header/.eslintignore]: Yes, you're right. For the other, there are some weird issues with the stencil-community plugin.

File: packages/components/tsconfig.eslint.json [hunk 4]
```diff
@@ -1,10 +1,5 @@
 {
   "extends": "./tsconfig.json",
-  "include": [
-    "src",
-    "**/tests"
-  ],
-  "exclude": [
-    "node_modules"
-  ]
+  "include": ["src", "**/tests"],
```
[the remark this answers is not in the document]
Comment (imagoiq) [reply packages/components/tsconfig.eslint.json:3]: I cannot reproduce it. Where does the error appear?

[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/components/tsconfig.eslint.json:3]: It only appears when the above suggestion is applied, so it will be fixed with #2167

**Review by gfellerph — APPROVED** [review]

**Pull request merged**